example-app/app/auth/effects/auth.effects.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { TestBed } from '@angular/core/testing';
+import { Actions } from '@ngrx/effects';
+import { RouterTestingModule } from '@angular/router/testing';
+import { cold, hot } from 'jasmine-marbles';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/observable/empty';
```

Comment:
Use `import { empty } from 'rxjs/observable/empty';` instead of patching the Observable